src/printer.rs

```diff
@@ -158,12 +159,17 @@ impl<'a> Printer<'a> {
         } else {
             for &(style, region) in regions.iter() {
                 let mut ansi_iterator = AnsiCodeIterator::new(region);
-                let mut ansi_prefix = "";
+                let mut ansi_prefix: String = String::new();
                 for chunk in ansi_iterator {
                     match chunk {
                         // ANSI escape passthrough.
                         (text, true) => {
-                            ansi_prefix = text;
+                            if text.chars().last().unwrap() == 'm' {
```

Comment:
Can we prevent the usage of `unwrap()` here? Maybe by using `.map_or(false, |c| c == 'm')`?